.github/workflow/community.yml

```diff
@@ -29,4 +29,6 @@ jobs:
       - uses: actions/checkout@v1
       - uses: EddieJaoudeCommunity/gh-action-community/src/statistics@main
         with:
-          firebase-key: ${{ secrets.FIREBASE_KEY }}
\ No newline at end of file
+          firebase-key: ${{ secrets.FIREBASE_KEY }}
+
+          
```

Comment:
We only need 1 new light. The red `-` is still there, probably because you have spaces on those links, they are not blank.